examples/logical_interconnects.py

```diff
@@ -231,3 +238,15 @@
     igmp_settings_updated = logical_interconnect.update_igmp_settings(igmp_settings)
     pprint(igmp_settings_updated)
     print("\nUpdated IGMP interconnect settings for the logical interconnect successfully")
+
+# Validates the bulk update from group operation and gets the consolidated inconsistency report
+if oneview_client.api_version >= 2000 and api_variant == 'Synergy':
+    print("Validates the bulk update from group operation and gets the consolidated inconsistency report")
+    bulk_validate_request = {
+        "logicalInterconnectUris": [
+            "/rest/logical-interconnects/d0432852-28a7-4060-ba49-57ca973ef6c2"
```

Comment:
instead of passing uri manually, better to do a GET call of LIs and use one Uri.
```
all_interconnects = logical_interconnects.get_all() # use any query params if required
use this - all_interconnects[0]['uri']
```

Reply:
I think that would create confusion to customers as request in APIdocs itself is having URIs in it, whereas your point brings sense for GET calls